### DISCUSSION TITLE
request to join team

### DISCUSSION THREAD

**Discussion by krishnaspeace** [ROM-builders/temporary#24531] [General]
Hello sir ,
I want to join you team
device tree - hhttps://gitlab.com/krishnaspeace/android_device_xiaomi_ysl/-/tree/evo
common tree - https://github.com/krishnaspeace/android_device_xiaomi_msm8953-common.git
vendor tree - https://github.com/krishnaspeace/vendor_xiaomi_ysl.git
kernel tree - https://github.com/krishnaspeace/android_kernel_xiaomi_msm8953.git
local manifests - https://github.com/krishnaspeace/local_manifests/blob/ysl/local_manifests.xml
telegram - https://t.me/krishanspeace
github - https://github.com/krishnaspeace
success build - https://cirrus-ci.com/task/5104131073900544

i hope now you accept my request

**Comment by ghost** [comment]
https://gitlab.com/krishnaspeace/android_device_xiaomi_ysl/-/commit/14e82c3b5a2b2d83369ecf7a8a518d1f0bd5d16c

Disappointing. Conflicts aren't fixed this way. They are fixed before push. This way you are just cherry-picking, ignoring the conflicts, and pushing to remote. Fix this up and push with proper way it is done

**Reply by krishnaspeace** [reply]
Does you are in team of rom builder

**Reply by ghost** [reply]
No
